networkx/algorithms/link_analysis/hits_alg.py

```diff
@@ -365,21 +365,19 @@ def hits_scipy(G, max_iter=100, tol=1.0e-6, nstart=None, normalized=True):
         return {}, {}
     M = nx.to_scipy_sparse_matrix(G, nodelist=list(G))
     (n, m) = M.shape  # should be square
-    A = M.T * M  # authority matrix
+    A = M.T @ M  # authority matrix
```

Comment:
I like that you are using A for the adjacency matrix above.  Maybe we should keep that convention.  Then I would replace
 ```
A = M.T @ M  # authority matrix	
```
with
```
ATA = A.T @ A # authority matrix	
```
Similarly,
```
AAT = A @ A.T # hub matrix	
```